data/net/minecraft/world/level/levelgen/feature/AbstractHugeMushroomFeature.mapping

```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/world/level/levelgen/feature/AbstractHugeMushroomFeature
+	METHOD getTreeHeight (Ljava/util/Random;)I
+		ARG 1 random
+	METHOD getTreeRadiusForHeight (IIII)I
+		ARG 3 foliageRadius
+	METHOD isValidPosition (Lnet/minecraft/world/level/LevelAccessor;Lnet/minecraft/core/BlockPos;ILnet/minecraft/core/BlockPos$MutableBlockPos;Lnet/minecraft/world/level/levelgen/feature/configurations/HugeMushroomFeatureConfiguration;)Z
+		ARG 1 level
+		ARG 2 pos
+		ARG 3 maxHeight
+		ARG 4 mutablePos
+		ARG 5 hugeMushroomFeatureConfiguration
```

Comment:
This one seems odd? That it wouldn't just be `configuration` a. la. other feature classes.